receiver/tlscheckreceiver/metadata.yaml

```diff
@@ -8,14 +8,13 @@ status:
   codeowners:
     active: [atoulme, michael-burt]
 
-
 resource_attributes:
-  tlscheck.url:
-    enabled: true
-    description: Url at which the certificate was accessed.
-    type: string
 
 attributes:
+  tlscheck.endpoint:
+    enabled: true
+    description: Endpoint at which the certificate was accessed.
+    type: string
```

Comment:
Why is this moved from resource attributes? It still seems like a resource attribute.